src/plus/github/githubGitProvider.ts

```diff
@@ -1,4 +1,5 @@
 /* eslint-disable @typescript-eslint/require-await */
+import type { HostingServiceType } from '@gitkraken/gitkraken-components';
```

Comment:
See above comment

Reply:
Done in commit: `Refactor: isolate the component imports to one file`.